custom_components/wyzeapi/__init__.py

```diff
@@ -6,49 +6,83 @@
 import logging
 import os
 import uuid
+from homeassistant import config_entries
 
 from homeassistant.config_entries import ConfigEntry, SOURCE_IMPORT
 from homeassistant.const import CONF_USERNAME, CONF_PASSWORD
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers import device_registry as dr
 from homeassistant.helpers.check_config import HomeAssistantConfig
 from wyzeapy import Wyzeapy
+from wyzeapy.wyze_auth_lib import Token
 
-from .const import DOMAIN, CONF_CLIENT
+from .token_manager import TokenManager
 
-PLATFORMS = ["light", "switch", "lock", "climate",
-             "alarm_control_panel"]  # Fixme: Re add scene
+from .const import DOMAIN, CONF_CLIENT, ACCESS_TOKEN, REFRESH_TOKEN, REFRESH_TIME
+
+PLATFORMS = [
+    "light",
+    "switch",
+    "lock",
+    "climate",
+    "alarm_control_panel",
+]  # Fixme: Re add scene
 _LOGGER = logging.getLogger(__name__)
 
 
 # noinspection PyUnusedLocal
-async def async_setup(hass: HomeAssistant, config: HomeAssistantConfig,
-                      discovery_info=None):
+async def async_setup(
+    hass: HomeAssistant, config: HomeAssistantConfig, discovery_info=None
+):
     # pylint: disable=unused-argument
     """Set up the Alexa domain."""
```

Comment:
Yikes I forgot this was still here. Can you tell where I got the boilerplate? lol

Reply:
I believe that entire function is uncessary, since I'm guessing that the config file isn't really the idea way to login (definitely isn't an option for 2fa)

Reply:
No I mean the comment that says """Set up the Alexa domain.""" lol

Reply:
Lol. Some copy/pasta I guess?